### ISSUE TITLE
 ♿ Flutter - Accessibility - Esmorga textfield is missing semantic label

### ISSUE THREAD

**Issue by ommacc** [CMMS-Esmorga/Esmorga-Project#4]
**Original Jira Issue:** [ESM-442](https://corunamobilemakers.atlassian.net/browse/ESM-442)

## Description

Semantic labels are used by screen readers to enable visually impaired users to
flutter: get spoken feedback about the contents of the screen and interact with the UI.

## Metadata

- **Type:** Bug
- **Status:** READY FOR DEV
- **Original Jira created:** 2026-02-25T17:46:05.531+0100
- **Priority:** Medium
- **Reporter:** QA Team
- **Labels:** Flutter

## Custom Fields

- **Component:** App - Frontend
- **Environment:** QA
- **Priority:** 🔵 Low
- **Steps to reproduce:** - Find Esmorga textfield in the app
- Use screen reader and navigate to the textfield
- **Rank:** 0|i002nz:
- **Development:** {}
